packages/create-platformatic-auto/src/index.mjs

```diff
@@ -13,24 +13,30 @@ import parseArgs from 'minimist'
 import ora from 'ora'
 import { createRequire } from 'node:module'
 import { pathToFileURL } from 'node:url'
+import { request } from 'undici'
 
 export async function chooseStackable (opts = {}) {
-  const choices = [
-    { name: 'Composer', value: '@platformatic/composer' },
-    { name: 'DB', value: '@platformatic/db' },
-    { name: 'Service', value: '@platformatic/service' }
-  ]
+  const stackablesRequest = request('https://marketplace.platformatic.dev/templates')
+  const stackablesRequestTimeout = new Promise((resolve, reject) => {
+    setTimeout(reject, 5000, new Error('Request timed out'))
```

Comment:
There is no need to throw an error here. Just write a simple function like that.

```js
async getDefaultStackables (timeout) {
  await setTimeout(timeout)
  return [
    '@platformatic/service',
    '@platformatic/db',
    '@platformatic/composer'
   ]
}
```

And then another function that fetches the stackables `fetchStackables` and also returns the list of stackable names.

Reply:
The call to fetch the stackables could fail though and that needs to be handled by returning the default list. 
By throwing the error after the timeout I can aggregate the logic of returning the default list in a single point.

Reply:
Make sense. Just move all the fetching + timeout logic to the separate function please. And check for the response status code before mapping the data.